app/notifications/callbacks.py

```diff
@@ -6,11 +8,13 @@
 
 
 def _check_and_queue_callback_task(notification):
+    if notification is None:
+        current_app.logger.warning("No notification provided, cannot queue callback task")
+        return
```

Comment:
A bit more null protection, I think I added not part of the scope of these changes and while pairing with someone.